PontoFacil/PontoFacil/ViewModels/SettingsPageViewModel.cs

```diff
@@ -26,36 +37,84 @@ public Profile Profile
         }
 
         public DelegateCommand SaveCommand { get; private set; }
-        #endregion
 
         private ISettingsService _settingsService;
 
+        private ResourceLoader resourceLoader;
+
+        private StringBuilder sbValidationMessages;
+        #endregion
+
+        #region Constructor
         public SettingsPageViewModel(ISettingsService settingsService)
         {
             _settingsService = settingsService;
 
+            resourceLoader = new ResourceLoader();
+
             Profile = _settingsService.GetProfile();
-            LunchTime = true;
-            if (Profile.LunchTime.Equals(2))
-                LunchTime = false;
+
+            LunchTimeOneHour = Profile.LunchTime == 1;
+            LunchTimeTwoHour = !LunchTimeOneHour;
 
             ApplicationView.GetForCurrentView().SetPreferredMinSize(new Size(500, 1000));
 
             InicializeCommands();
         }
+        #endregion
 
+        #region Commands
         private void InicializeCommands()
         {
             SaveCommand = new DelegateCommand(ActionSave);
         }
 
-        private void ActionSave()
+        private async void ActionSave()
```

Comment:
Async method should return a Task.

Reply:
Do not return Task when method is call by delegate command.